ArduPlane/Plane.h

```diff
@@ -1163,7 +1163,7 @@ class Plane : public AP_Vehicle {
     void dspoiler_update(void);
     void airbrake_update(void);
     void landing_neutral_control_surface_servos(void);
-    void servos_output(void);
+    void servos_output(bool cork = true);
```

Comment:
Thanks for this.  It's not my area but it might be safer to remove the default from the argument and force the caller to make a choice.. maybe add a comment to help them make the choice.  Just an idea though

Reply:
I agree, the default hides the other users (here on github anyway) which are important to understand the scope of the change.